addons/stock/tests/test_move2.py

```diff
@@ -2375,8 +2377,9 @@ class TestStockUOM(TestStockCommon):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        dp = cls.env.ref('product.decimal_product_uom')
-        dp.digits = 7
+        with cls.sudo():
```

Comment:
rather than using with cls.sudo. Do it with an admin user or create a function to add the admin right for the stock user